server/src/application/application.entity.ts

```diff
@@ -47,4 +48,7 @@ export class Application extends CustomBaseEntity {
 
   @ManyToOne(() => User, (user) => user.applications)
   user: User;
+
+  @OneToMany(() => File, (file) => file.application)
+  files: File[];
```

Comment:
I'm wondering if `File` is a specific enough name? Would something like `SupportingDocument` be better? What do you think?

Reply:
yeah can be renamed. Makes sense!